packages/reactive-element/src/reactive-element.ts

```diff
@@ -837,7 +835,6 @@ export abstract class ReactiveElement
     // connection promise
     if (this.__enableConnection) {
       this.__enableConnection();
-      this.__pendingConnectionPromise = this.__enableConnection = undefined;
```

Comment:
Is `__enableConnection` even set anymore? Can you remove it?

Reply:
Removed, was dead code.